### PR TITLE
Issue/#195 Fix coop scores not getting posted to leaderboard

### PR DESCRIPTION
**Pull request by Askaholic**
The LIKE query was not matching `filename` column correctly. `filename` has the format 'maps/name.version.zip' which is the same format as self.game.map_file_path. So the old query was checking for something like: '%/maps/name.version.zip.%' which clearly doesn't match the entries in the db.

The test will pass if it the db contains an entry for the prothyon coop map, however, this won't be available to Travis until a new DB release is made and faf-stack is updated to use it. But feel free to test it locally with the latest test-data.sql :D

### CODE REVIEW

**Comment by Askaholic** [comment]
#195

**Comment by Rackover** [comment]
May be related to https://github.com/FAForever/server/issues/372

**Comment by Sheeo** [comment]
Good catch  👍

**Comment by speed2CZ** [comment]
If this actually works, can someone wipe out the current times?

**Comment by Rackover** [comment]
The current times ?

**Comment by speed2CZ** [comment]
All the replays that are currently in the leaderboards, they are acient, broken, outdated

File: tests/unit_tests/test_gameconnection.py [hunk 1]
```diff
@@ -147,16 +152,35 @@ def test_abort(game_connection, game, players):
...
+
 async def test_handle_action_GameResult_draw_ends_sim(game, game_connection):
     game_connection.ConnectToHost = CoroMock()
     await game_connection.handle_action('GameResult', [0, 'draw'])
 
     assert game_connection.finished_sim
     assert game.check_sim_end.called
+
+
+async def test_handle_action_OperationComplete(game, game_connection):
```
Comment (norraxx) [inline tests/unit_tests/test_gameconnection.py]: IMHO we should write what do we test. Yes, we can talk about right name of the test, but we won't explain to everybody what test really does and why.

File: tests/unit_tests/test_gameconnection.py [hunk 2]
```diff
@@ -147,16 +152,35 @@ def test_abort(game_connection, game, players):
...
+    pytest.skip("requires test data from faf-db:v63 which doesn't exist yet")
+
+    game.map_file_path = "maps/prothyon16.v0005.zip"
+    secondary = 1
+    time_taken = '09:08:07.654321'
+    await game_connection.handle_action('OperationComplete', ['1', secondary, time_taken])
+
+    import server.db as db
```
Comment (norraxx) [inline tests/unit_tests/test_gameconnection.py]: don't we have fixture for db connection? If not, i wish that we would have it.

Comment (Rackover) [reply tests/unit_tests/test_gameconnection.py]: We have a fixture for the db_pool in tests/conftest.py

**Pull request merged**